Assets/Scripts/Door/Door.cs

```diff
@@ -0,0 +1,85 @@
+using Mirror;
+using UnityEngine;
+
+public class Door : NetworkBehaviour
+{
+    [SyncVar] [SerializeField] private DoorState _DoorState = DoorState.Closed;
```

Comment:
@DP94 Does [SyncVar] indicate it's something that's to be shared between players in multiplayer? :)

Reply:
https://mirror-networking.gitbook.io/docs/guides/synchronization/syncvars